larsim/EventGenerator/SingleGen_module.cc

```diff
@@ -173,10 +173,11 @@ namespace evgen {
         Name("ThetaXzYzHist"),
         Comment("name of the histograms of angular (X-Z and Y-Z) distribution"),
         [this]() { return fromHistogram(AngleDist()); }};
-
-      fhicl::OptionalAtom<rndm::NuRandomService::seed_t> Seed{
+      
+      fhicl::Atom<rndm::NuRandomService::seed_t> Seed{
         Name("Seed"),
-        Comment("override the random number generator seed")};
+        Comment("override the random number generator seed"),
+        0 /* dummy? */};
```

Comment:
@mantheys, based on @PetrilloAtWork's comment, I suggest:

```suggestion
        rndm::NuRandomService::InvalidSeed};
```

With that change, I would consider this PR ready.

Reply:
Done!